react/features/speaker-stats/reducer.ts

```diff
@@ -85,31 +90,26 @@ function _updateCriteria(state: ISpeakerStatsState, { criteria }: { criteria: st
  * @returns {Object} - The new state after the reduction of the specified action.
  */
 function _updateStats(state: ISpeakerStatsState, { stats }: { stats: any; }) {
-    const finalStats = state.pendingReorder ? stats : state.stats;
-
-    if (!state.pendingReorder) {
-        // Avoid reordering the speaker stats object properties
-        const finalKeys = Object.keys(stats);
-
-        finalKeys.forEach(newStatId => {
-            finalStats[newStatId] = _.clone(stats[newStatId]);
-        });
-
-        Object.keys(finalStats).forEach(key => {
-            if (!finalKeys.includes(key)) {
-                delete finalStats[key];
-            }
-        });
-    }
+    return {
+        ...state,
+        stats
+    };
+}
 
-    return _.assign(
-        {},
-        state,
-        {
-            stats: { ...finalStats },
-            pendingReorder: false
-        }
-    );
+/**
+ * Reduces a specific Redux action UPDATE_SORTED_SPEAKER_STATS_IDS of the feature speaker-stats.
+ *
+ * @param {Object} state - The Redux state of the feature speaker-stats.
+ * @param {Action} action - The Redux action UPDATE_SORTED_SPEAKER_STATS_IDS to reduce.
+ * @private
+ * @returns {Object} The new state after the reduction of the specified action.
+ */
+function _updateSortedSpeakerStats(state: ISpeakerStatsState, { participantIds }: { participantIds: Array<string>; }) {
```

Comment:
It seems the previous logic was to preserved the sorted state of the object, I think we can update our redux store without modifying the stats coming from LJM